### PR TITLE
fix(docs): document the new flow events

### PR DESCRIPTION
**Pull request by philbooth**
I forgot all about these when reviewing #1409, sorry.

@vbudhram, r?

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -53,35 +52,81 @@ The activity events respect the `dnt` "Do not track" header and do not send `utm
...
-|`device.created`|`event`, `userAgent`, `time`, `uid`, `device_id`|`flow_id`, `flow_time`, `context`, `entrypoint`, `migration`, `service`, `utm_campaign`, `utm_content`, `utm_medium`, `utm_source`, `utm_term`|
-|`device.updated`|`event`, `userAgent`, `time`, `uid`, `device_id`|`flow_id`, `flow_time`, `context`, `entrypoint`, `migration`, `service`, `utm_campaign`, `utm_content`, `utm_medium`, `utm_source`, `utm_term`|
-|`device.deleted`|`event`, `userAgent`, `time`, `uid`, `device_id`|`context`, `entrypoint`, `migration`, `service`, `utm_campaign`, `utm_content`, `utm_medium`, `utm_source`, `utm_term`|
+|`account.created`|`event`, `userAgent`, `time`, `flow_id`, `flow_time`|`context`, `entrypoint`, `migration`, `service`, `utm_campaign`, `utm_content`, `utm_medium`, `utm_source`, `utm_term`|
+|`account.verified`|`event`, `userAgent`, `time`, `flow_id`, `flow_time`|`context`, `entrypoint`, `migration`, `service`, `utm_campaign`, `utm_content`, `utm_medium`, `utm_source`, `utm_term`|
+|`account.login`|`event`, `userAgent`, `time`, `flow_id`, `flow_time`|`context`, `entrypoint`, `migration`, `service`, `utm_campaign`, `utm_content`, `utm_medium`, `utm_source`, `utm_term`|
```
Comment (vbudhram) [inline docs/overview.md]: Noticed that columns with `flow_id` and `flow_time` are not sorted, should they be? Is this to call them out?

**Comment by vbudhram** [comment]
@philbooth just the one q, otherwise LGTM, r+

File: docs/overview.md (hunk 1 above)
Comment (philbooth) [inline docs/overview.md]: Good point, I didn't consider sorting. Will fix.

**Comment by vladikoff** [comment]
Yaaaay!

**Pull request merged**